### PR TITLE
feat: add type definitions

### PR DESCRIPTION
**Pull request by potato4d**
## About

Write type definition.

## Changes

- [x] Install `@types/execa`
- [x] Write `types/lmify.d.ts`

## Note

It may be better to test with TypeScript to ensure type integrity.

### CODE REVIEW

File: types/lmify.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as execa from 'execa'
+
+declare class Lmify {
+
+  // for initalize
+  init(): void
+  _init(): Promise<void>
```
Comment (pi0) [inline types/lmify.d.ts]: I think would be a better idea to not expose internals

**Review by pi0 — COMMENTED** [review]
Thanks, @potato4d!

**Comment by potato4d** [comment]
@pi0 
thank you your review.
resolved.

**Comment by potato4d** [comment]
@pi0 
Is there anything else I need to do?

**Comment by pi0** [comment]
Thank you @potato4d. Sorry, my notifications were so messy and missed your comment :(

Types are available in 0.2.1.

**Comment by potato4d** [comment]
@pi0 
I was a little worried about my problem with the fix. lol
Thank you for merging!

**Pull request merged**